Automattic/wp-calypso
# Social Logins: Fix social buttons style

Related to pdDR7T-Z4-p2#comment-1155

## Proposed Changes
This pull request https://github.com/Automattic/wp-calypso/pull/82602 tried to fix `invite-accept` styles but end up breaking the form in two places [here](https://wordpress.com/start/videopress-account/user/en?variationName=videopress&flow=videopress&pageTitle=Video%20Portfolio&redirect_to=/setup/videopress/options) and [here](https://wordpress.com/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Dc43d84a8c7fdeb15b6523e926ad27f24b24f1514093b181ab27529e223fa5606%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252Fstart%25252F%252523%25252Finstallation%2526source%253Dhero_cta%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=50916&bypassTargetRedirection=true).

This PR is fixing it by:

- Moving the styles related to `invite-accept` into the invite-accept component
- Removing unnecessary style that were breaking `videopress` and woo from `sigupform`


## Testing Instructions
- Use live link or pull and run this branch


**To validate that these changes won't break anything, please navigate using incognito to:**

- /setup/newsletter
- /setup/domain-transfer/


These should be fixed:

-  `/start/videopress-account/user/en?variationName=videopress&flow=videopress&pageTitle=Video%20Portfolio&redirect_to=/setup/videopress/options`

- `/start/wpcc/oauth2-user?ref=oauth2&oauth2_redirect=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%2F%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Dc43d84a8c7fdeb15b6523e926ad27f24b24f1514093b181ab27529e223fa5606%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253D%25252Fstart%25252F%252523%25252Finstallation%2526source%253Dhero_cta%26blog_id%3D0%26wpcom_connect%3D1%26wccom-from%26calypso_env%3Dproduction%26jetpack-code%26jetpack-user-id%3D0%26action%3Doauth2-login%26from-calypso%3D1&oauth2_client_id=50916&bypassTargetRedirection=true`
## Pre-merge Checklist
